calico-enterprise/networking/configuring/node-local-dns-cache.mdx

```diff
@@ -0,0 +1,227 @@
+---
+description: Configure Calico Enterprise for NodeLocal DNSCache
+---
+
+# Configure Calico Enterprise for NodeLocal DNSCache
+
+## Big picture
+
+Configure Calico Enterprise for clusters with NodeLocal DNSCache feature provided by Kubernetes on versions 1.18+. 
```

Comment:
I think you can remove "on versions 1.18+", since we don't support 1.18 any longer. Our users can read our k8s compatibility here (1.24+):  https://docs.tigera.io/calico-enterprise/latest/getting-started/compatibility